generators/app/index.js

```diff
@@ -542,7 +542,7 @@ module.exports = class extends BaseBlueprintGenerator {
                     this.configOptions.skipComposeEntity = true;
                     const options = this.options;
                     const configOptions = this.configOptions;
-                    this.getExistingEntities().forEach(entity => {
+                    this.getExistingEntities(true).forEach(entity => {
```

Comment:
This makes little sense to have a Boolean parameter for such a function... Plus, it's a code-smell. Does it deserve its own function?

Reply:
See bellow response.
It’s possible to remove this parameter and always set the file as modified on mem-fs.
I was removing overhead, there isn’t much.